src/componentes/login.js

```diff
@@ -8,15 +8,20 @@ import { Link } from "react-router-dom";
 import { loginSchema } from "../schemas/schema";
```

Comment:
change `login.js` to `Login.js`  component names and file should be in PascalCase

Reply:
Done!